assets/src/styles/partials/_parameter-list.scss

```diff
@@ -0,0 +1,92 @@
+/*
+ * Styles related to the Parameter Selection List.
+ */
+@import '../components/hydrograph/variables';
+
+$row-border-color: 'black';
+
+.select-time-series-container {
+  #parameter-selection-title {
+    @include u-font-weight('bold');
+    @include u-font-size('body', 'lg');
+    @include u-margin-bottom(2px);
+  }
+
+  #select-time-series {
+    @include grid-container;
+    @include u-margin(0);
+    @include u-padding(0);
+  }
+
+  .main-parameter-selection-container {
+    .selected {
+      background-color: $selected;
+    }
+
+    .grid-row-period-of-record {
+      @include grid-row;
+      .grid-row-period-of-record-text {
+        @include grid-col(10);
+        @include grid-offset(1);
+      }
+      .toggle-for-top-period-of-record {
+        @include grid-col(1);
+      }
+      @include at-media('tablet') {
+        display: none;
+      }
+    }
+
+    .grid-row-container-row {
+      @include grid-container;
+      @include u-border-bottom(1px, $row-border-color);
+      @include u-padding(1);
+    }
+
+    .grid-row-container-row:first-child {
+      @include u-border-top(1px, $row-border-color);
+    }
+
+    .grid-row-inner {
+      .usa-radio__label {
+        @include u-margin-top(0);
+        @include u-margin-bottom(0);
+      }
+      .description__param-select {
+        @include grid-col(7);
+        @include u-font-weight('bold');
+      }
+      @include at-media('tablet') {
+        .description__param-select {
+          @include u-font-weight('normal');
+        }
+      }
+    }
+
+    .grid-row-inner {
```

Comment:
Seems unnecessary to add a second .grid-row-inner section. Could you move the contents up to the section above?

Reply:
Darn it. I made a note to myself to do that, and then got distracted and missed it. Thanks for pointing it out.